src/cli.ts

```diff
@@ -201,12 +202,21 @@ if (program.cacheValuesets && !program.vsApiKey) {
 
 const cacheDirectory = 'cache/terminology';
 
+const parseTrustMetaProfile = (value?: string): boolean => {
+  if (value === undefined) return true; // if flag no flag
```

Comment:
I can't tell if this is a typo or a quirky comment 😂 fine either way but maybe `if no flag default true` could be good

Reply:
typo, im just dumb lol